carbonmark/components/pages/Project/index.tsx

```diff
@@ -50,9 +51,11 @@ const Page: NextPage<PageProps> = (props) => {
   }
 
   const allListings =
-    Array.isArray(project.listings) && getAllListings(project.listings);
+    Array.isArray(project.listings) &&
+    getAllListings(project.listings as Listing[]);
```

Comment:
When you give the `listingsGetters`like
- getActiveListings
- getAllListings
- getSortByUpdateListings
- ....
the correct type `Listing without Project` then you can remove the assertions here, right?

Reply:
cleaned it all up thx!